CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ These changes are available on the `master` branch, but have not yet been releas
   `Permissions.use_external_sounds` and
   `Permissions.view_creator_monetization_analytics`.
   ([#2620](https://github.com/Pycord-Development/pycord/pull/2620))
+- Added the ability to use functions with any number of optional arguments, and
+  functions returning an awaitable as `Option.autocomplete`
+  ([#2669](https://github.com/Pycord-Development/pycord/pull/2669)).
```

Comment:
```suggestion
- Added the ability to use functions with any number of optional arguments, and
  functions returning an awaitable as `Option.autocomplete`.
  ([#2669](https://github.com/Pycord-Development/pycord/pull/2669))
```

Reply:
Because of https://discord.com/channels/881207955029110855/881735314987708456/1315946233117937664

Reply:
Keep as-is for now (dot before paranthesis), we can switch with a later version.